syntax = "proto3";

message MuAds1259Store {
  uint32 reading = 1;
}
